.github/workflows/end_to_end_tests.yaml

```
@@ -0,0 +1,107 @@
+# (C) Copyright 2025, SECO Mind Srl
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: End to End tests
+
+on: [pull_request]
+env:
+  ASTARTE_API_HOSTNAME: api.autotest.astarte-platform.org
+  ASTARTE_BROKER_HOSTNAME: broker.autotest.astarte-platform.org
+  ASTARTE_ACTION_CERT: /usr/local/share/ca-certificates/astarte-autotest.crt
+  QEMU_GATEWAY_ADDR: 10.0.2.2
+
+jobs:
+  end-to-end-test:
+    runs-on: ubuntu-latest
+    concurrency:
+      group: end-to-end-test-${{ github.ref }}
+      cancel-in-progress: true
+    # container: espressif/idf:release-v5.4
```

Remove this, if it is no longer used
```suggestion
    # container: espressif/idf:release-v5.4
```